app/helpers/application_helper.rb

```diff
@@ -122,8 +122,12 @@ def volunteer_links(event)
     end, ', ')
   end
 
-  def event_types_sentence(conference)
-    conference.event_types.map { |et| et.title.pluralize }.to_sentence
+  def event_types_sentence(conference, is_admin)
```

Comment:
Hmm, I think since this method just controls the public/displayed text, we actually don't need to special case the admin features.

Reply:
can you explain what you mean by special case.